src/TokenRepository.php

```diff
@@ -15,7 +15,7 @@ class TokenRepository
      */
     public function create($attributes)
     {
-        return Token::create($attributes);
+        return config('passport.token.model', Laravel\Passport\Token::class)::create($attributes);
```

Comment:
I would suggest making a method that returns a new token instance... like `newTokenInstance`. So you don't have to repeat this config call so many times in this file.